actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkUploadBackgroundJobActor.java

```diff
@@ -132,32 +133,56 @@ private void processUser(BulkUploadProcessTask task, String organisationId) {
             task, ProjectUtil.BulkProcessStatus.FAILED, ex.getMessage(), userMap, JsonKey.CREATE);
         return;
       }
-      if (userMap.get(JsonKey.ORG_ID) != null) {
-        orgMap = getOrg((String) userMap.get(JsonKey.ORG_ID));
-        if (orgMap == null) {
+      if (StringUtils.isNotBlank((String) userMap.get(JsonKey.ORG_ID))
```

Comment:
Instead of reading the values several times from the hash map. Store in local variables and use it. It makes code more readable.